packages/ui-design-system/src/FiltersBar/internals/NumberValueFilter.tsx

```diff
@@ -23,10 +23,10 @@ export function NumberValueFilter({
   const [opSelectIsOpen, setOpSelectIsOpen] = useState(false);
   const [localValue, setLocalValue] = useState<NumberComparisonFilter>(
     (() => {
-      const sv = filter.selectedValue ?? { operator: 'eq', value: 0 };
-      const raw = (sv as any).value as unknown;
+      const sv = filter.selectedValue ?? { op: '=', value: 0 };
```

Comment:
Some variable naming makes it hard to read, `sv` could easily be named `selectedValue`